glean-core/android/src/main/java/mozilla/telemetry/glean/scheduler/PingUploadWorker.kt

```diff
@@ -130,15 +51,6 @@ class PingUploadWorker(context: Context, params: WorkerParameters) : Worker(cont
     companion object {
         internal const val PING_WORKER_TAG = "mozac_service_glean_ping_upload_worker"
 
-        // NOTE: The `PINGS_DIR` must be kept in sync with the one in the Rust implementation.
-        internal const val PINGS_DIR = "pending_pings"
```

Comment:
Just wondering, but should we have tests in the platforms that touch things that are really only messed with by the core? This could totally be a follow up, but I wanted to know your opinion.

For me it doesn't make sense to do that, [the whole assertion](https://github.com/mozilla/glean/blob/ef37b8d11a7ef22ead7d0c1be83971abd2b2155e/glean-core/android/src/test/java/mozilla/telemetry/glean/GleanTest.kt#L103-L107) on weather the ping was saved to the pending_pings folder on the tests that use this `PINGS_DIR` constant could be dropped. We could verify that only baseline was submitted by checking that only one request was sent out, for example, and even that is arguable.

Reply:
> 
> 
> Just wondering, but should we have tests in the platforms that touch things that are really only messed with by the core? This could totally be a follow up, but I wanted to know your opinion.

Yes, I think that's a great idea... for a follow-up!

> For me it doesn't make sense to do that, [the whole assertion](https://github.com/mozilla/glean/blob/ef37b8d11a7ef22ead7d0c1be83971abd2b2155e/glean-core/android/src/test/java/mozilla/telemetry/glean/GleanTest.kt#L103-L107) on weather the ping was saved to the pending_pings folder on the tests that use this `PINGS_DIR` constant could be dropped. We could verify that only baseline was submitted by checking that only one request was sent out, for example, and even that is arguable.

Yes, I agree. On the application side, we do need a way to check that only one ping is received, though. Let's file a bug about this and take it as a follow-up.

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1620672